mlir/tools/mlir-opt/mlir-opt.cpp

```diff
@@ -274,6 +274,10 @@ int main(int argc, char **argv) {
   DialectRegistry registry;
   registerAllDialects(registry);
   registerAllExtensions(registry);
+
+  // TODO: Remove this and the corresponding MLIRToLLVMIRTranslationRegistration
+  // cmake dependency when a safe dialect interface registration mechanism is
+  // implemented, see D157703 (and corresponding note on the declaration).
```

Comment:
Why this TODO for removing this? I don't get the reference to D157703?

Reply:
I copied/adapted the TODO from the declaration that this is calling. It looked like this was all intended to be temporary and I wanted to make sure that when this line was removed that someone also removes the corresponding dep in CMake. I could take out the reference.

Reply:
(actually looking at the declaration makes more sense)